htdocs/accountancy/journal/bankjournal.php

```diff
@@ -970,8 +971,8 @@ function writebookkeeping() {
 				print $reflabel;
 				print "</td>";
 				print "<td>" . $val["type_payment"] . "</td>";
-				print "<td align='right'>" . ($mt >= 0 ? price($mt) : '') . "</td>";
-				print "<td align='right'>" . ($mt < 0 ? price(- $mt) : '') . "</td>";
+				print "<td align='right' nowrap>" . ($mt >= 0 ? price($mt) : '') . "</td>";
```

Comment:
To add a nowrap use
class="nowraponall"  (no wrap on all devices)
or
class="nowrap"  (no wrap on desktop only)